opm/simulators/flow/KeywordValidation.hpp

```diff
@@ -121,9 +125,30 @@ namespace KeywordValidation
         const UnsupportedKeywords m_keywords;
         const PartiallySupportedKeywords<std::string> m_string_items;
         const PartiallySupportedKeywords<int> m_int_items;
+        const PartiallySupportedKeywords<double> m_double_items;
     };
 
 
+    // Helper class to test if a given value is with a list of permitted values.
+    template <typename T>
+    class permit
+    {
+    public:
+        permit(const std::initializer_list<T>& permitted_values) {
+            for(auto item: permitted_values) {
+                m_permitted.push_back(item);
+            }
+        }
+
+        bool operator()(const T value) const
```

Comment:
I guess the argument could be a `const T&`?

Reply:
yes. done.